app/package.json

```diff
@@ -20,7 +20,8 @@
     "bs-css": "^9.0.1",
     "react": "^16.8.1",
     "react-dom": "^16.8.1",
-    "reason-react": ">=0.7.0"
+    "reason-react": ">=0.7.0",
+    "reductive": "https://github.com/MargaretKrutikova/reductive#hooks"
```

Comment:
Do you think we should perhaps clone the repo on our side, just in case this PR disappears without getting merged?